easybuild/easyconfigs/f/FastQC/FastQC-0.11.8.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'PackedBinary'
+
+name = 'FastQC'
+version = '0.11.8'
+versionsuffix = '-Java-%(javaver)s'
```

Comment:
@fizwit Because of the `versionsuffix`, the correct filename is `FastQC-0.11.8-Java-1.8.eb`, please fix?

Reply:
fixed